lib/fluent/plugin/in_snmptrapalert.rb

```diff
@@ -93,6 +93,10 @@ def start
                                 trap_events[vb.name.to_s] = vb.value.to_s
                             end
                             trap_events['host'] = trap.source_ip
+                            if trap.kind_of?(SNMP::SNMPv1_Trap)
```

Comment:
While we are here, add also the generic_trap

Reply:
done